servers/rendering_server.cpp

```diff
@@ -2306,6 +2306,10 @@ RenderingServer::RenderingServer() {
 					"rendering/vulkan/rendering/back_end",
 					PROPERTY_HINT_ENUM, "ForwardClustered,ForwardMobile"));
 
+	GLOBAL_DEF("rendering/shader_compiler/shader_cache/enabled", true);
+	GLOBAL_DEF("rendering/shader_compiler/shader_cache/compress", true);
+	GLOBAL_DEF("rendering/shader_compiler/shader_cache/strip_debug_on_release", true);
```

Comment:
Remember that we have [feature tags](https://docs.godotengine.org/en/stable/getting_started/workflow/export/feature_tags.html), which may be a more suitable way to conditionally enable/disable the feature in a non-hardcoded way.

Reply:
Thanks, I forgot we had a release tag